lib/src/base-styles/_nx-page.scss

```diff
@@ -23,13 +22,12 @@
 
 .nx-page-title__page-icon {
   display: inline-block;
-  font-size: 26px;
+  font-size: 32px;
```

Comment:
Couldn't this just be removed?

Reply:
> Couldn't this just be removed?

Yes it can: https://github.com/sonatype/sonatype-react-shared-components/pull/162/commits/83899fab8464f5dabce9f2c20e57244a11366456